### PR TITLE
Add missing libraries to README.md

### PR DESCRIPTION
**Pull request by stephentoub**

### CODE REVIEW

**Comment by jasonwilliams200OK** [comment]
:+1: 
On a related note, can we also have this repo updated: https://github.com/dotnet/corefx-progress/blob/master/src-diff/README.md? Currently it is showing Done = 67, Coming = 53. I think the real stats are bit better. :smile:

**Comment by stephentoub** [comment]
> can we also have this repo updated: https://github.com/dotnet/corefx-progress/blob/master/src-diff/README.md?

@terrajobst?  @ellismg?

File: README.md [hunk 1]
```diff
@@ -92,6 +92,7 @@ More libraries are coming soon (the overall list of items we currently plan to m
...
 |**System.Reflection**<br/>[![MyGet Package](https://img.shields.io/myget/dotnet-core/v/System.Reflection.svg)](https://www.myget.org/gallery/dotnet-core)|<sub>Provides types that retrieve information about assemblies, modules, members, parameters, and other entities in managed code by examining their metadata.</sub>|
 |**System.Reflection.DispatchProxy**<br/>[![MyGet Package](https://img.shields.io/myget/dotnet-core/v/System.Reflection.DispatchProxy.svg)](https://www.myget.org/gallery/dotnet-core)|<sub>Provides a mechanism for dynamically creating proxy types that implement a specified interface and derive from a specified DispatchProxy type.</sub>|
+|**System.Reflection.Emit.ILGeneration**<br/>[![MyGet Package](https://img.shields.io/myget/dotnet-core/v/System.Reflection.Emit.ILGeneration.svg)](https://www.myget.org/gallery/dotnet-core)|<sub>Provides clases that allow a compiler or other tool to emit Microsoft intermediate language (MSIL).</sub>|
```
Comment (justinvp) [inline README.md]: Typo: "clases" -> "classes". Though, "types" may be better.

Comment (stephentoub) [inline README.md]: Thanks, will fix. I copied this text directly from https://www.nuget.org/packages/System.Reflection.Emit.ILGeneration/, where the typo apparently also exists :)

**Pull request merged**